src/integrationTest/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/allocatethecase/CaseWorkerServiceTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit.allocatethecase;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import java.io.IOException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.iacaseapi.domain.handlers.presubmit.allocatecase.CaseWorkerName;
+import uk.gov.hmcts.reform.iacaseapi.domain.service.CaseWorkerService;
+import uk.gov.hmcts.reform.iacaseapi.domain.service.IdamService;
+
+@SpringBootTest
+@ActiveProfiles("integration")
+public class CaseWorkerServiceTest {
+
+    @Autowired
+    private WireMockServer caseWorkerRefDataServiceMock;
```

Comment:
There is common way of using wiremock in inregration tests in case-api, it would be good to keep it consistent if in case we need some refactoring, with JUnit 5 it can be injected with annotation:
```
public void adds_a_case_note(
        @WiremockResolver.Wiremock(factory = StaticPortWiremockFactory.class) WireMockServer server)
```